web/js/app.js

```diff
@@ -71,6 +71,31 @@ function ControladorEditar($scope, $http, $location, $routeParams) {
   }
 }
 
+/* Controlador para agregar usuarios */
+function ControladorAgregar($scope, $http, $location, $routeParams) {
```

Comment:
__[eslint]__
* Error - "$routeParams" is defined but never used (no-unused-vars)